addons/web/static/src/views/list/list_arch_parser.js

```diff
@@ -69,6 +69,7 @@ export class ListArchParser extends XMLParser {
         let buttonGroup;
         let handleField = null;
         let defaultOrder = stringToOrderBy(xmlDoc.getAttribute("default_order") || null);
+        const defaultGroupBy = xmlDoc.getAttribute("default_group_by");
```

Comment:
attributes of the root node are retrieved below (~ line 200), maybe move this there and use `treeAttr` for the sake of consistency

Reply:
Hello @aab-odoo ,
Do you want I move the defaultOrder as well? It is up to you? 🙂 

Thanks for your review. 🙂